applications/SolversApplication/SolversApplication.py

```diff
@@ -0,0 +1,10 @@
+from KratosSolversApplication import *
+application = KratosSolversApplication()
+application_name = "KratosSolversApplication"
+application_folder = "SolversApplication"
+
+# The following lines are common for all applications
+from .. import application_importer
```

Comment:
please use the new style for importing applications(as python-modules only), see the template-application 

This will make things easier when Kratos moves to this entirely